specificatie/genereervariant/openapi.yaml

```diff
@@ -20,9 +20,9 @@ paths:
     get:
       tags:
       - Beheren volgindicaties
-      summary: Raadplegen alle actieve volgindicaties
+      summary: Raadpleeg actieve volgindicaties
       description: |
-        Opvragen van alle actuele (actieve) volgindicaties van een abonnee. Volgindicaties met een einddatum in het verleden worden niet geleverd.
+        Vraag de actieve volgindicaties op van een abonnee. Levert geen volgindicaties met als einddatum vandaag of een einddatum in het verleden.
```

Comment:
overal veranderen in: Levert geen volgindicaties met einddatum vandaag of in het verleden.